marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubProjectImpl.java

```diff
@@ -324,15 +326,27 @@ public void upgradeProject() throws IOException {
         Path oldEntityConfigDir = this.projectDir.resolve("entity-config.old");
         Path oldHubInternalConfigDir = this.projectDir.resolve("hub-internal-config.old");
         Path oldUserConfigDir = this.projectDir.resolve("user-config.old");
+        
+        //can be changed to traverse the old hub-internal-config and build the set 
```

Comment:
I think this is good, I think it's helpful to document all of the files that we expect from 3.x